src/main/resources/migration/sql/001_Basic_tables.sql

```diff
@@ -0,0 +1,68 @@
+CREATE SCHEMA messaging;
+CREATE TABLE IF NOT EXISTS messaging.person
+(
+    person_id uuid NOT NULL,
+    name character varying(50) NOT NULL,
+    date_of_birth timestamp with time zone,
+    person_info character varying(300),
+    CONSTRAINT person_table_pk PRIMARY KEY (person_id)
+);
+COMMENT ON TABLE messaging.person IS 'Данные пользователя';
+COMMENT ON COLUMN messaging.person.person_info IS 'Информация в вольном стиле от пользователя о себе';
+CREATE TABLE IF NOT EXISTS messaging.dialog
+(
+    message_id uuid NOT NULL,
+    dialog_id uuid NOT NULL,
+    writer_id uuid NOT NULL,
+    reader_id uuid NOT NULL,
+    create_time timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
As a frequently using option, we can consider a `created_at` name

Reply:
yep, you're right